config/samples/backends/lvm-nvme-tcp/lvm.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+    service: cinder
+    component: cinder-volume
+  name: 99-master-cinder-lvm-losetup
+spec:
+  config:
+    ignition:
+      version:  3.2.0
+    Systemd:
+      Units:
+      - name: cinder-lvm-losetup.service
+        enabled: true
+        Contents:  |
+          [Unit]
+          Description=Cinder LVM losetup
+          DefaultDependencies=no
+          Conflicts=umount.target
+          Requires=lvm2-monitor.service systemd-udev-settle.service
+          Before=local-fs.target umount.target
+          After=var.mount lvm2-monitor.service systemd-udev-settle.service
+
+          [Service]
+          Environment=VG_GB_SIZE=10G
+          Environment=LOOPBACK_FILE=/var/home/core/cinder-volumes
+          Type=oneshot
+          ExecStart=bash -c "if [[ ! -e ${LOOPBACK_FILE} ]]; then /bin/truncate -s ${VG_GB_SIZE} ${LOOPBACK_FILE} && /sbin/vgcreate cinder-volumes `/sbin/losetup --show -f ${LOOPBACK_FILE}`; else /sbin/losetup -f ${LOOPBACK_FILE}; fi"
+          ExecStop=bash -c "/sbin/losetup -d `/sbin/losetup -j ${LOOPBACK_FILE} -l -n -O NAME`"
+          RemainAfterExit=yes
+
+          [Install]
+          WantedBy=local-fs-pre.target
```

Comment:
The new linter they will introduce in pr 147 will complain about the lack of newline at end of file.

Reply:
Fixed